nixos/modules/services/mail/dovecot.nix

```diff
@@ -376,6 +367,26 @@ in
       description = lib.mdDoc "Quota limit for the user in bytes. Supports suffixes b, k, M, G, T and %.";
     };
 
+
+    pluginSettings = mkOption {
+      # types.str does not coerce from packages, like `sievePipeBinScriptDirectory`.
+      type = types.attrsOf (types.oneOf [ types.str types.package ]);
+      default = {};
+      example = literalExpression ''
+        {
+          sieve = "file:~/sieve;active=~/.dovecot.sieve";
+        }
+      '';
```

Comment:
```suggestion
      example = {
        sieve = "file:~/sieve;active=~/.dovecot.sieve";
      };
```

:)

Reply:
I think this is kind of weird to have inconsistency on this in the file, see:
```
      example = literalExpression ''
        {
          Spam = { specialUse = "Junk"; auto = "create"; };
        }
      '';
```

I did it for the list, but it's weird to do it for the attribute set. What's the harm?

Reply:
There isn't, it's just not necessary if it's just a simple expression that can't be evaluated any further afaik :P Keep it if you want.

Reply:
I prefer to keep it this way and if we want to migrate this out, feel free to send follow up PRs? :p